packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -30,17 +30,14 @@ class _FlutterDemoState extends State<FlutterDemo> {
         center: new Text('Flutter Demo')
       ),
       body: new Center(
-        child: new Text(
-          'Button tapped $_counter times.',
-          key: const ValueKey('counter')
```

Comment:
I don't think we should junk up the main example code with extra stuff for the driver test.